src/main/java/org/deepsymmetry/Wayang.java

```diff
@@ -265,4 +276,51 @@ public static void sendFrame() {
             }
         }
     }
+
+    /**
+     * Send a frame of pixels async, corresponding to whatever has been drawn in the image returned by open(),
+     * to the display.
+     *
+     * @throws LibUsbException       if there is a problem communicating.
+     * @throws IllegalStateException if the Push 2 has not been opened.
+     */
+    public static void sendFrameAsync() {
+        if (transferBuffer == null) {
+            throw new IllegalStateException("Push 2 device has not been opened");
+        }
+        IntBuffer transferred = IntBuffer.allocate(1);
+        transfer = LibUsb.allocTransfer();
+        LibUsb.fillBulkTransfer(transfer, pushHandle, (byte) 0x01, headerBuffer, LibUsb::freeTransfer,  null, 1000);
+
+        int result = LibUsb.submitTransfer(transfer);
+        if (result != LibUsb.SUCCESS) {
+            throw new LibUsbException("Transfer of frame header to Push 2 display failed", result);
+        }
+
+        // We send 8 lines at a time to the display; allocate buffers big enough to receive them,
```

Comment:
This looks like a copy-paste; we are no longer sending 8 lines at a time. But this should probably just say “many” so we can adjust `LINES_PER_TRANSFER` without worrying about fixing all the comments.